store/src/java-test/com/zimbra/cs/index/event/logger/InMemoryEventLoggerTest.java

```diff
@@ -0,0 +1,39 @@
+package com.zimbra.cs.index.event.logger;
+
+import com.zimbra.common.service.ServiceException;
+import com.zimbra.cs.account.Account;
+import com.zimbra.cs.account.Provisioning;
+import com.zimbra.cs.index.event.Event;
+import com.zimbra.cs.mailbox.Mailbox;
+import com.zimbra.cs.mailbox.MailboxManager;
+import com.zimbra.cs.mailbox.MailboxTestUtil;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.Collections;
+import java.util.HashMap;
+
+public class InMemoryEventLoggerTest {
+    private static Provisioning prov;
+    private static Account acct;
+    private static Mailbox mbox;
+
+    @BeforeClass
+    public static void init() throws Exception {
+        MailboxTestUtil.initServer();
+        prov = Provisioning.getInstance();
+        acct = prov.createAccount("test@zimbra.com", "test123", new HashMap<>());
```

Comment:
For ease of unit test diagnostics, we tend to give test accounts descriptive names - in this case, something like "inMemoryEventLoggerTest@zimbra.com"